website/docs/user_guide/unleash-context.md

```diff
@@ -17,14 +17,15 @@ _All fields are optional_, but some strategies depend on certain fields being pr
 
 The below table gives a brief overview over what the fields' intended usage is, their lifetime, and their type. Note that the exact type can vary between programming languages and implementations. Be sure to consult your specific client SDK for more information on its implementation of the Unleash Context.
 
-| field name        | type                  | lifetime | description                            |
-|-------------------|-----------------------|----------|----------------------------------------|
-| `appName`         | `string`              | static   | the name of the application            |
-| `environment`[^1] | `string`              | static   | the environment the app is running in  |
-| `userId`          | `string`              | dynamic  | an identifier for the current user     |
-| `sessionId`       | `string`              | dynamic  | an identifier for the current session  |
-| `remoteAddress`   | `string`              | dynamic  | an identifier for the current session  |
-| `properties`      | `Map<string, string>` | dynamic  | a key-value store of any data you want |
+| field name        | type                  | lifetime | description                                                                                | introduced in |
+|-------------------|-----------------------|----------|--------------------------------------------------------------------------------------------|---------------|
+| `appName`         | `string`              | static   | the name of the application                                                                |               |
+| `environment`[^1] | `string`              | static   | the environment the app is running in                                                      |               |
+| `userId`          | `string`              | dynamic  | an identifier for the current user                                                         |               |
+| `sessionId`       | `string`              | dynamic  | an identifier for the current session                                                      |               |
+| `remoteAddress`   | `string`              | dynamic  | an identifier for the current session                                                      |               |
+| `properties`      | `Map<string, string>` | dynamic  | a key-value store of any data you want                                                     |               |
+| `currentTime`[^2]     | `DateTime`/`string`   | special / read-only  | A `DateTime` (or similar) data class instance or a string in an RFC3339-compatible format. |    v4.9                                  |
```

Comment:
I don't know whether this field is writable at all or how it works. Can the field be set by the user or is it read-only? Can the user access it whenever they want?

Reply:
It can be set by the user but all the SDKs will default this to DateTime.now or whatever the language equivalent is